src/Symfony/Bridge/Twig/NodeVisitor/TranslationDefaultDomainNodeVisitor.php

```diff
@@ -51,8 +53,8 @@ public function enterNode(Node $node, Environment $env): Node
                 return $node;
             } else {
                 $var = $this->getVarName();
```

Comment:
I just discovered that this node visitor breaks reproducible compilation here.

Reply:
good point, let's fix this explicitly in a separate PR

Reply:
I just noticed that this was fixed for 7.2 in #57609. Should we backport that PR?

Reply:
that PR is actually broken

Reply:
let's backport (and fix the broken part) in 5.4 yes
reproducible builds FTW

Reply:
A way to generate a reproducible name that does not clash with parent or child template could be to use a hash of the template name in the variable name (with the new hardcoded name as prefix)

Reply:
see #58706